### PR TITLE
Retain 2FA authorization

### LINKED ISSUE #15351: Two factor auth setting is not retained after modifying some other config
### Describe the bug

Set up two factor authentication in PHPMyAdmin and then change a setting. See that 2FA is not enabled any more.

### To Reproduce

1. Log in
2. Setup 2FA
2. Logout
3. Login with 2FA enabled
4. Go to setting, functions and toggle version checks
5. See that 2FA is not enabled any more

### Expected behavior

2FA should still be enabled.

### Server configuration

- Operating system: Debian 9.8
- Web server: Apache 2.4
- Database version: MariaDB 10.1.38
- PHP version: 7.3.6
- phpMyAdmin version: 4.9.0.1

### Client configuration

- Browser: Firefox 67.0.4
- Operating system: Win7 x64

### PR DESCRIPTION
**Pull request by victorphoenix3**
Fixes #15351 .
Signed-off-by: Jayati Shrivastava <gaurijove@gmail.com>

## BEFORE
before

## AFTER
after

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Review by williamdes — APPROVED** [review]
LGTM

@victorphoenix3 is it ready ?
I will test it soon

**Comment by victorphoenix3** [comment]
@williamdes  Yes, the PR seems to fix the bug ,do test it.
But i haven't yet been able to find WHY the set value for the 2fa backend gets lost.

**Comment by williamdes** [comment]
> But i haven't yet been able to find WHY the set value for the 2fa backend gets lost.

Okay, you know your fix looks like he issue #15724 fixed by #15735 :thinking:

**Comment by williamdes** [comment]
My guess is that the 2FA settings are not loaded from the database

**Comment by victorphoenix3** [comment]
> My guess is that the 2FA settings are not loaded from the database

@williamdes  oh, I'll try to debug and pin down the cause.
PR is ready for review.

**Comment by williamdes** [comment]
I am also trying to debug actually ^^
I found out that userPreferences is loaded with 2fa value but is not saved with 2fa value

This first one who finds posts it here :)

**Comment by williamdes** [comment]
Done, I used your fix

**Comment by williamdes** [comment]
@victorphoenix3 can you review so I can merge quickly ?

File: test/classes/Plugins/Auth/AuthenticationCookieTest.php [hunk 1]
```diff
@@ -1018,6 +1018,19 @@ public function testCookieDecrypt()
                 'sec321'
             )
         );
+        $this->assertEquals(
```
Comment (williamdes) [inline test/classes/Plugins/Auth/AuthenticationCookieTest.php]: never mind, just added some unrelated tests ^^

**Comment by williamdes** [comment]
Thank you for your contribution @victorphoenix3 !

**Pull request merged**